aos/closed_loop/calcZernikesFromDonutStamps.ipynb

```diff
@@ -8,10 +8,10 @@
     "# CalcZernikes from DonutStamps\n",
```

Comment:
<p>I notice given the data paths that the previous version of this notebook used different dataset.  Is this only a more recent imsim simulation, or is it phosim vs imsim?  Either way would be good to mention here where does the dataset come from. Or is this notebook meant to be understood as coming in a sequence? If so, perhaps we rename them or mention that explicitly in the objective (eg. notebook meant to use after <code>cutOutDonutsWithWep.ipynb</code> ..  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I added a short data section to the logistics section below describing that this is imSim data and the outputs are from a cutout task described in the <code>cutOutDonutsWithWep.ipynb</code> notebook. </p>